qulice-checkstyle/src/main/resources/com/qulice/checkstyle/checks.xml

```diff
@@ -344,7 +344,7 @@
         </module>
         <module name="LineLength">
             <property name="max" value="80"/>
-            <property name="ignorePattern" value="^import .*$"/>
+            <property name="ignorePattern" value="^import .*$|https?:\/\/.+"/>
```

Comment:
@krzyk That is true and I thought about that. However, perhaps it would be beneficial to skip lines such as this as well:

```java
private final String ZK_URL = "http://some.url/atSomePathEtcEtcEtcEtcEtc"
```

It would be rather difficult to trip up the pattern http:// without it being in a javadoc or string literal. Also, the original requestor did state "I'd expect all lines containing http:// or https:// to be skipped from this check."